lib/gtfs/components/StopMarker.js

```diff
@@ -21,9 +20,46 @@ type Props = {
 
 export default class StopMarker extends Component<Props> {
   _onClick = () => {
-    const {newEntityId, onStopClick, stop} = this.props
+    const {onStopClick, stop} = this.props
     const {feed} = stop
-    onStopClick && onStopClick(stop, feed, newEntityId)
+    if (onStopClick) onStopClick({feed, entities: [stop]})
+    else console.warn('No action defined for onClick.')
+  }
+
+  _getChildStopsContent = () => {
+    const {popupAction, stop} = this.props
+    if (!stop.child_stops || stop.child_stops.length === 0) return null
+    return (
+      <div className='well' style={{marginTop: '5px'}}>
+        <strong>
+          NOTE: Stop is parent station to {stop.child_stops.length} stop(s):
+        </strong>
+        <ul style={{paddingLeft: '15px'}}>
+          {stop.child_stops.map((s, i) =>
+            <li key={i}>
+              {s.stop_name} ({s.stop_code ? s.stop_code : s.stop_id})
+            </li>
+          )}
+        </ul>
+        <Button
+          bsStyle='warning'
+          block
+          onClick={this._onClickChildren}>
+          <Icon type='map-marker' /> {popupAction} all stops
```

Comment:
Suggestion to rename this variable to `popupActionPrefix` for better code understandability.